src/mctp-netlink.c

```diff
@@ -1067,6 +1091,12 @@ static int linkmap_add_entry(mctp_nl *nl, struct ifinfomsg *info,
 		return -1;
 	}
 
+	if (ifaddr_len > MAX_ADDR_LEN) {
+		warnx("linkmap, too long ifaddr '%*s'", (int)ifaddr_len,
+		      ifaddr);
```

Comment:
`ifaddr` is not a string

Reply:
I wondered if I should print a hex representation of the address here, but I think as opposed to the text representation of `ifname`, the length for the address would probably be enough to track any issues.